### DISCUSSION TITLE
area calculation with GOS smaller than 2 degree

### DISCUSSION THREAD

**Discussion by pxyg** [mtex-toolbox/mtex#1442] [Ask Anything]
Dear all,

Recently I learnt that GOS value can be somehow used as an indicator for recrystallization. when GOS is smaller than 2 degree, the corresponding area can be treated as the recrystallized area.   

I searched the previous issues, but I did not find an issue about area calculation for recrystallization (maybe I missed somewhere else). Can someone give some tipps for this?  It is easy to get the GOS map from the documentation, but I could not figure out how to calculate the area fraction.

Thanks!!

Best regards
Zhendong

**Comment by kilir** [comment]
Hi,
the GOS is the average orientation spread in a grain, so it sort of contains the area. It it is indeed a proxy for recrystallization might depend on your field.
Cheers,
Rüdiger

**Comment by Liujianhua00** [comment]
I think you just need a logical comparison, like a=grains('phase').GOS./degree<2;

**Comment by Achintya-Patra** [comment]
I think you can partition out the grains with GOS less that 2 degrees as suggested by @Liujianhua00 and then just take the ratio of number of pixels in the partition to the total number of pixels in the original file.

drx = grains((grains('phase').GOS./degree<2))
